src/TimerOutput.jl

```diff
@@ -5,15 +5,17 @@ mutable struct TimeData
     ncalls::Int
     time::Int64
     allocs::Int64
+    firstexec::Int64
 end
-
+TimeData(ncalls, time, allocs) = TimeData(ncalls, time, allocs, time)
```

Comment:
Perhaps this needs typedefs? I didn't add them as `time_ns()` returns `UInt64` but time is `Int64` so wasn't sure what to do